<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>498</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Room Monitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/resources/resources/logo_img.svg</normaloff>:/resources/resources/logo_img.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="MonitoringPlotView" name="monitoringPlot"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAnalysis"/>
    <addaction name="separator"/>
    <addaction name="actionChiudi"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Modifica</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Aiuto</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuVisualizza">
    <property name="title">
     <string>Visualizza</string>
    </property>
    <addaction name="actionGestione_Monitoraggio"/>
    <addaction name="actionUltimi_MAC_Rilevati"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuVisualizza"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string/>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAnalysis"/>
   <addaction name="actionSettings"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="monitoringManagerDock">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="monitoringLabel">
         <property name="text">
          <string>Gestione Monitoraggio</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>Start</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/resources/resources/play.svg</normaloff>:/resources/resources/play.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="text">
            <string>Stop</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/resources/resources/stop.svg</normaloff>:/resources/resources/stop.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="lastMacDock">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="macSituationLabel">
       <property name="text">
        <string>Ultime posizioni MAC rilevati</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="macSituation"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/resources/resources/settings.svg</normaloff>:/resources/resources/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Impostazioni...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/resources/resources/info.svg</normaloff>:/resources/resources/info.svg</iconset>
   </property>
   <property name="text">
    <string>Informazioni</string>
   </property>
  </action>
  <action name="actionChiudi">
   <property name="text">
    <string>Chiudi</string>
   </property>
  </action>
  <action name="actionGestione_Monitoraggio">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gestione Monitoraggio</string>
   </property>
  </action>
  <action name="actionUltimi_MAC_Rilevati">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ultimi MAC Rilevati</string>
   </property>
  </action>
  <action name="actionAnalysis">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/resources/resources/analysis.svg</normaloff>:/resources/resources/analysis.svg</iconset>
   </property>
   <property name="text">
    <string>Analisi dei dati</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MonitoringPlotView</class>
   <extends>QGraphicsView</extends>
   <header>windows/elements/MonitoringPlotView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>startButton</tabstop>
  <tabstop>stopButton</tabstop>
 </tabstops>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionChiudi</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>monitoringManagerDock</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionGestione_Monitoraggio</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGestione_Monitoraggio</sender>
   <signal>toggled(bool)</signal>
   <receiver>monitoringManagerDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUltimi_MAC_Rilevati</sender>
   <signal>triggered(bool)</signal>
   <receiver>lastMacDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastMacDock</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionUltimi_MAC_Rilevati</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
